src/Backend/Modules/Pages/Actions/Edit.php

```diff
@@ -556,6 +585,9 @@ protected function parse()
 
         // parse the tree
         $this->tpl->assign('tree', BackendPagesModel::getTreeHTML());
+
+        // assign if profiles module is installed
+        $this->tpl->assign('authentication', BackendModel::isModuleInstalled('Profiles'));
```

Comment:
I would name this something like `showAuthenticationTab` since it better describes the purpose, or `profilesModuleIsInstalled`